code/part-two/processor/services/prng.js

```diff
@@ -2,9 +2,9 @@
 
 /**
  * A function that takes any hex string as a seed, and returns a pseudo-random
- * number generator. This function will return a new seemingly random number
- * every time it is called, but will be identical to any other generator
- * created with the same seed.
+ * number generator. This generator function will return a semi-random number
+ * every time it is called. But the numbers generated will be identical to
+ * those from another generator created with the same seed.
```

Comment:
**For discussion:**

I like how Wikipedia mentions _sequence_ and _reproducibility_: https://en.wikipedia.org/wiki/Pseudorandom_number_generator

Thoughts on this wording? (Maybe it's too much?)
> This generator function will return a (seemingly) random number every time it is called. However, the exact same sequence of (seemingly) random numbers can be reproduced from another generator function created with the same seed, making this a _pseudo_ random number generator instead of a truly random number generator.